rbac/internal/views.py

```diff
@@ -73,11 +80,44 @@
 )
 from api.utils import RESOURCE_MODEL_MAPPING, get_resources
 
+# environment variables
+HOST = os.getenv("HOST")
+client_id = os.getenv("CLIENT_ID")
+client_secret = os.getenv("CLIENT_SECRET")
+scopes = os.getenv("SCOPES")
+url = os.getenv("URL")
+grant_type = os.getenv("GRANT_TYPE")
+relations_api_server = os.getenv("relation_api_gRPC_server")
 
 logger = logging.getLogger(__name__)
 TENANTS = TenantCache()
 PROXY = PrincipalProxy()
 
+# Create redis client
+redis_client = redis.StrictRedis(host="localhost", port=6379, decode_responses=True)
```

Comment:
this runs when server but we can use existing login for it, I think it needs just import related redis classes, please take a look how redis cache is used here. look on TenantCache class ,..